packages/core/src/runtime-configurations.ts

```diff
@@ -1,11 +1,5 @@
 import { IRunOptions, type TopLevelConfig } from './types.js';
 
-// eslint-disable-next-line @typescript-eslint/no-namespace
-declare namespace globalThis {
```

Comment:
This was a tricky thing to do...I didn't want the cast `(globalThis as unknown as Window)` in isMainEntrypoint because it's not true this code can also run in node.